### PR TITLE
Add HURUmap Ethiopia

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

Add data from HURUmap ET to the Takwimu HURUmap instance

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots
screencapture-localhost-8000-profiles-country-et-ethiopia-2019-01-30-10_29_49

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/templates/profile/profile_detail_takwimu.html [hunk 1]
```diff
@@ -311,4 +342,4 @@ <h2 class="header-for-columns">Government Spending</h2>
     </article>
   {% endif %}
 
-{% endblock %}
+{% endblock %}
```
Comment (kilemensi) [inline takwimu/templates/profile/profile_detail_takwimu.html]: Missing new line.

**Comment by phillipahereza** [comment]
> 👍 @phillipahereza. But:
> 
> Don't we already have population files in Takwimu where the other countries put their data? Shouldn't Ethiopia data go there as well?
> 
> `population_residence.sql`
> `population_sex.sql`

@kilemensi the Ethiopia files are disaggregated by more variables e.g. `gender_age` has age range which is not in `population_sex.sql`

**Comment by kilemensi** [comment]
> > +1 @phillipahereza. But:
> > Don't we already have population files in Takwimu where the other countries put their data? Shouldn't Ethiopia data go there as well?
> > `population_residence.sql`
> > `population_sex.sql`
> 
> @kilemensi the Ethiopia files are disaggregated by more variables e.g. `gender_age` has age range which is not in `population_sex.sql`

Cool @phillipahereza.

It's my understanding we're going to be adding more disaggregated data moving forward so how are going to keep track of these two types of data (`population_sex.sql` for aggregated data and `gender_age.sql` for the disaggregated data)?

I'd prefer if we come up with a system that:

 1. Keeps files with similar data in similarly named files e.g. `population_sex.sql`and `populaton_sex_disaggregated.sql`  for aggregated and disaggregated data on population by sex respectively, and
 2. Easy to scan and know which are aggregated/disaggregated.

But it's your call.

**Comment by phillipahereza** [comment]
@kilemensi I thought about it, and I figured we could just look for extra data for the other countries so that it's similar to what we have for Ethiopia

**Comment by kilemensi** [comment]
> @kilemensi I thought about it, and I figured we could just look for extra data for the other countries so that it's similar to what we have for Ethiopia

So how does one know `gender_age_group.sql` is about population and it's somewhat the disaggregated version of `population_sex.sql`?

**Comment by phillipahereza** [comment]
> > @kilemensi I thought about it, and I figured we could just look for extra data for the other countries so that it's similar to what we have for Ethiopia
> 
> So how does one know `gender_age_group.sql` is about population and it's somewhat the disaggregated version of `population_sex.sql`?

@kilemensi lemme update the names

**Comment by kilemensi** [comment]
@phillipahereza :

 1. Add sources, and
 2. We now have two population sex/gender vizs where one says Female population is **50%** and the other **51%**. Lets stick to just one (the most recent) and lets stick to `sex`. Gender is what a person identify themselves or social roles based on their sex. Hence the files should `population_sex_age_group.sql` and `population_sex_urban_rural.sql` or `population_sex_residence.sql`.

**Pull request closed without merging**